### PR TITLE
November bitdevs

### PR DESCRIPTION
**Pull request by lorenzolfm**

### CODE REVIEW

File: _posts/2024-11-08-socratic-seminar-30.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+### Warmup
+
+* [Building Bitcoin - Funding a $1.2 Trillion Dollar Project](http://1a1z.com/fund.html)
+* [Send Bitcoin to anyone that has an email address. (It uses MPC)](https://emailbtc.net/)
+
+## Bitcoin
+
+* [Disclosure of CVE-2024-35202](https://bitcoincore.org/en/2024/10/08/disclose-blocktxn-crash/)
+* [Disclosure of CVE-2024-35202](https://bitcoincore.org/en/2024/10/08/disclose-blocktxn-crash/)
```
Comment (storopoli) [inline _posts/2024-11-08-socratic-seminar-30.md:28]: Duplicated

File: _posts/2024-11-08-socratic-seminar-30.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+* [CVE-2024-38365 public disclosure (btcd `FindAndDelete` bug)](https://delvingbitcoin.org/t/cve-2024-38365-public-disclosure-btcd-findanddelete-bug/1184?u=antoinep)
+* [Disclosure of DoS due to inv-to-send sets growing too large](https://bitcoincore.org/en/2024/10/08/disclose-large-inv-to-send)
+* [Disclosure of hindered block propagation due to mutated blocks](https://bitcoincore.org/en/2024/10/08/disclose-mutated-blocks-hindering-propagation)
+* [Guide for Wallets Employing Bitcoin Core 28.0 Policies](https://bitcoinops.org/en/bitcoin-core-28-wallet-integration-guide/)
+* [Deprecated RPC on Bitcoin Core v28.0 prevents LND from starting](https://x.com/roasbeef/status/1842956355516223824)
+* [Bitcoin Core v28.0 released](https://github.com/bitcoin/bitcoin/releases/tag/v28.)
```
Comment (storopoli) [inline _posts/2024-11-08-socratic-seminar-30.md:34]: Broken link:
```suggestion
* [Bitcoin Core v28.0 released](https://github.com/bitcoin/bitcoin/releases/tag/v28.0)
```

File: _posts/2024-11-08-socratic-seminar-30.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+* [What specific verifications are done on a fresh Bitcoin TX and in what order?](https://bitcoin.stackexchange.com/questions/124221/what-specific-verifications-are-done-on-a-fresh-bitcoin-tx-and-in-what-order)
+
+### Lightning
+
+* [LN Summit 2024 summary](https://delvingbitcoin.org/t/ln-summit-2024-notes-summary-commentary/1198)
+* [Phoenix moved its LSP to a new open protocol with an open-source implementation](https://x.com/PhoenixWallet/status/1844377194489053555)
+
+### Cryptography
+
+* [MuSig2 merged into secp256k1 repo](https://github.com/bitcoin-core/secp256k1/pull/1479)
```
Comment (storopoli) [inline _posts/2024-11-08-socratic-seminar-30.md:49]: Also released on version 0.6.0 yesterday

File: _posts/2024-11-08-socratic-seminar-30.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+### Mining
+
+* [OCEAN - DATUM](https://ocean.xyz/docs/datum)
+* [MDK Beta Update: Early Feedback & Expanded Participation](https://www.mining.build/blog/mdk-beta-update/)
+    - [www.mining.build](https://www.mining.build/manuals/)
+
```
Comment (storopoli) [inline _posts/2024-11-08-socratic-seminar-30.md:56]:
```suggestion

### Layer 2 ++

* [Alpen Labs launches vanilla bitcoin signet with a BitVM-based bridge and Zero-knowledge commitments on-chain](https://www.stratabtc.org)
```

Can I do a shameless plug?

**Review by storopoli — CHANGES_REQUESTED** [review]
Some fixes, and a shameless plug.

File: _posts/2024-11-08-socratic-seminar-30.md (hunk 1 above)
Comment (lorenzolfm) [reply _posts/2024-11-08-socratic-seminar-30.md]: tks, removed

File: _posts/2024-11-08-socratic-seminar-30.md (hunk 4 above)
Comment (lorenzolfm) [reply _posts/2024-11-08-socratic-seminar-30.md:56]: added that for you

Comment (lorenzolfm) [reply _posts/2024-11-08-socratic-seminar-30.md:56]: [adding storopoli shameless plug](https://github.com/saopaulobitdevs/saopaulobitdevs.org/pull/39/commits/ee85b036ffe7dbf364b688f231b6dbe97548a591)

File: _posts/2024-11-08-socratic-seminar-30.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+### R&D
+
+* [P2QRH (Pay To Quantum Resistant Hash) BIP draft](https://github.com/bitcoin/bips/pull/1670)
+* [Shielded CSV](https://mailing-list.bitcoindevs.xyz/bitcoindev/b0afc5f2-4dcc-469d-b952-03eeac6e7d1b@gmail.com/)
+* [Bitcoin PIPEs: Introducting Covenants without Soft Forks](https://www.allocin.it/uploads/placeholder-bitcoin.pdf)
+* [Unlocking Mass-Market Self-Custody: Secure and Private Smartphone Bitcoin Wallets](https://bitkey.build/unlocking-mass-market-self-custody-3/)
+    - [assets.ctfassets.net](https://assets.ctfassets.net/mtmp6hzjjvnd/6Qjcs8zgMiyffC0Uk8cx4V/6d1553946b50b132209518be8ff3026f/Unlocking_Mass_Market_Self_Custody_-10-24-.pdf?ref=bitkey.build)
+* [Relatório sobre ecash da HRF]([https://mailchi.mp/hrf.org/hrfs-weekly-financial-freedom-report-9100630?e=[UNIQID]](https://mailchi.mp/hrf.org/hrfs-weekly-financial-freedom-report-9100630?e=%5BUNIQID%5D))
```
Comment (brunoerg) [inline _posts/2024-11-08-socratic-seminar-30.md:64]: Talvez deixar em inglês pq vai ser em ingles essa edição?
```suggestion
* [Report about ecash from HRF]([https://mailchi.mp/hrf.org/hrfs-weekly-financial-freedom-report-9100630?e=[UNIQID]](https://mailchi.mp/hrf.org/hrfs-weekly-financial-freedom-report-9100630?e=%5BUNIQID%5D))
```

Comment (oleonardolima) [reply _posts/2024-11-08-socratic-seminar-30.md:64]: O markdown tambem parece nao interpretar corretamente o link 🤔.

**Pull request merged**